livre/eq/Config.cpp

```diff
@@ -194,16 +199,22 @@ bool Config::init( const int argc, char** argv )
 
 uint32_t Config::frame()
 {
+    if( _impl->dataFrameRange == INVALID_FRAME_RANGE )
+        return eq::Config::finishFrame();
```

Comment:
Do you need the frame number returned? If no, just make frame() void and return; here?